src/dirs.proj

```diff
@@ -4,6 +4,7 @@
 
   <ItemGroup>
     <Project Include="*\**\*.csproj" />
+    <Project Include="*\**\*.depproj" />
```

Comment:
Here's a sample from my yet-to-be-merged branch that overlaps with a bit of the work you were doing: https://github.com/ericstj/buildtools/commit/55c433fe6a764fb6bdd16851ba2e74512885becf